### DISCUSSION TITLE
Streets/POIs in "Famagusta within the Walls" barely visible

### DISCUSSION THREAD

**Discussion by FlSc** [mapsforge/mapsforge#1324] [General]
First: thanks for all the effort in providing the maps!

I found an issue, using the Cyprus Map as of 2022-03-03 08:57 from https://ftp-stud.hs-esslingen.de/pub/Mirrors/download.mapsforge.org/maps/v5/europe/cyprus.map
I open the map with c:geo (version 2022.06.06) on two different Android devices and the area of "Famagusta within the Walls" is grey. Only few street or POI names are visible (see screen shot).
I would expect that all streets, POIs, etc. are visible.

I don't know whether this is an conversion issue in Mapsforge, an issue with c:geo or this is a general OSM data issue. However on [OSM](https://www.openstreetmap.org/#map=16/35.1244/33.9433) issue is not visible.
Perhaps someone could double check!

Thanks a lot!
![Famagusta_Issue](https://user-images.githubusercontent.com/12628520/172894078-6901cb73-fe10-42ed-88d9-a7c64b1ab288.png)

**Comment by devemux86** [comment]
Map rendering depends on:
- Map content ([Mapsforge tag-mapping](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-map-writer/src/main/config/tag-mapping.xml))
- Map style ([Mapsforge themes](https://github.com/mapsforge/mapsforge/tree/master/mapsforge-themes/src/main/resources/assets/mapsforge))

e.g. it could be a polygon rendered over the map elements.

**Comment by devemux86** [comment]
It is the [Osmarender](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-themes/src/main/resources/assets/mapsforge/osmarender.xml) map style (z-order of rules).
With the [Default](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-themes/src/main/resources/assets/mapsforge/default.xml) map style the rendering is better:

![Cruiser](https://user-images.githubusercontent.com/3484020/172897569-0ef56714-3855-41ab-a7d2-58f07243175c.png)
